src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,50 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        List<String> listData;
+        try {
+            File file = new File(fromFileName);
+            listData = Files.readAllLines(file.toPath());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read data from file: " + fromFileName, e);
+        }
+        File file = new File(toFileName);
+        try {
+            file.createNewFile();
+        } catch (IOException e) {
+            throw new RuntimeException("Can't create file: " + toFileName, e);
+        }
+        try {
+            Files.write(file.toPath(), calculatedData(listData).getBytes());
+        } catch (IOException e) {
```

Comment:
Let's extract this lines to the method `writeToFile()`.
Two try catch blocks are not needed. Combine the logic in one `try catch` please.